### PR TITLE
My First Article

### PR DESCRIPTION
**Pull request by harshadyeola**

### CODE REVIEW

**Comment by harshadyeola** [comment]
@MiteshShah Please review and publish

**Comment by MiteshShah** [comment]
@harshadyeola I'll check and merge this soon.

File: _posts/devops/2016-04-17-docker-getting-started.markdown [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: Docker - Getting Started
+author: Harshad_Yeola
+modified:
+comments: true
+categories: devops
```
Comment (MiteshShah) [inline _posts/devops/2016-04-17-docker-getting-started.markdown]: Its should be devops/docker

File: _posts/devops/2016-04-17-docker-getting-started.markdown [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+excerpt:
+tags: [devops]
```
Comment (MiteshShah) [inline _posts/devops/2016-04-17-docker-getting-started.markdown]: tags: [devops, docker]

**Comment by MiteshShah** [comment]
@harshadyeola When people click on DevOps https://miteshshah.github.io/devops/
There should be Docker is listed
For that edit https://github.com/MiteshShah/miteshshah.github.io/blob/master/devops/index.md

Once Docker is listed there when we click Docker its return 404
For that create docker directory at https://github.com/MiteshShah/miteshshah.github.io/tree/master/devops

cd docker and create index.md file which include brief what we covers in Docker
i.e Docker Installation (your first article) 

The just add text what you cover and when you write that add links.

Sample index.md for my ansible

```
---
layout: page
title: "Ansible"
tags: [Linux, Ansible, IT Automation, DevOps]
modified: 22016-03-30T16:20:56+05:30
comments: true
image:
  feature: https://cloud.githubusercontent.com/assets/1223371/14140416/6f3e0fe2-f696-11e5-90dc-43cbe773b92a.jpeg
  credit:
  creditlink:
---

{% include _toc.html %}
{% include _subscribe.html %}

### Installation


<li><a href="/devops/ansible/how-to-install-ansible/#mac-os-x"> Install Ansible on Mac OS </a></li>
<li><a href="/devops/ansible/how-to-install-ansible/#ubuntu"> Install Ansible on Ubuntu </a></li>
<li><a href="/devops/ansible/how-to-install-ansible/#redhatcentos"> Install Ansible on RedHat/CentOS </a></li>
```

I prefer you can take a look into https://miteshshah.github.io/linux/elk/ and https://github.com/MiteshShah/miteshshah.github.io/tree/master/linux/elk

File: _posts/devops/2016-04-17-docker-getting-started.markdown [hunk 3]
```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: Docker - Getting Started
```
Comment (MiteshShah) [inline _posts/devops/2016-04-17-docker-getting-started.markdown]: @harshadyeola Change title to Install Docker
Also change file name to 2016-04-17-install-docker.markdown

As we publish series of tutorials so its better we use SEO friendly name as people mostly search how to install xyz

File: _posts/devops/2016-04-17-docker-getting-started.markdown (hunk 2 above)
Comment (harshadyeola) [inline _posts/devops/2016-04-17-docker-getting-started.markdown]: @MiteshShah do you mean `_posts/devops/docker/2016-04-17-docker-getting-started.markdown`?

**Comment by harshadyeola** [comment]
@MiteshShah Okay I will do this

File: _posts/devops/2016-04-17-docker-getting-started.markdown (hunk 2 above)
Comment (MiteshShah) [inline _posts/devops/2016-04-17-docker-getting-started.markdown]: @harshadyeola Change tags: [devops] to tags: [devops, docker]
At present we dont have search post by tag but soon we add it so when people want to find docker post we give them on one shot not display all devops.

When people want to search devops we display all devops tags

**Comment by harshadyeola** [comment]
what about this?

```
layout: post
title: Docker - Installation
author: Harshad_Yeola
modified:
comments: true
categories: devops
excerpt:
tags: [devops, docker]
image:
  url:
  alt: Docker - Installation
  title: Docker - Installation
  feature:
date: 2016-04-17T23:02:54+05:30
```

**Comment by MiteshShah** [comment]
@harshadyeola categories: devops/docker
except above all looks great :)

**Comment by harshadyeola** [comment]
@MiteshShah Please check now

File: _posts/devops/docker/2016-04-17-docker-installation.markdown [hunk 4]
```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: Docker - Installation
+author: Harshad_Yeola
+modified:
+comments: true
+categories: devops/docker
+excerpt:
```
Comment (MiteshShah) [inline _posts/devops/docker/2016-04-17-docker-installation.markdown]: @harshadyeola when you open https://miteshshah.github.io/
Every post has small description what is post.

excerpt: "Step by step guide to install Docker on Ubuntu, Debian & CentOS"

**Comment by MiteshShah** [comment]
@harshadyeola Last suggestion https://github.com/MiteshShah/miteshshah.github.io/pull/3#discussion_r60272764 and we are good to merge :)

**Comment by harshadyeola** [comment]
@MiteshShah  now ?

**Comment by MiteshShah** [comment]
@harshadyeola One bug image block return 404.
https://github.com/MiteshShah/miteshshah.github.io/commit/908e5bdcc01f2de5000f836fab126a53edd2a33f

File: devops/index.md [hunk 5]
```diff
@@ -11,7 +11,4 @@ image:
 ---
 
 
-{% for post in site.categories.devops %}
```
Comment (MiteshShah) [inline devops/index.md:14]: @harshadyeola Dont delete anything please
All you need to add your new content.

**Pull request merged**